Source/WebCore/platform/graphics/gbm/GBMBufferSwapchain.cpp

```diff
@@ -45,7 +45,9 @@ GBMBufferSwapchain::~GBMBufferSwapchain() = default;
 
 RefPtr<GBMBufferSwapchain::Buffer> GBMBufferSwapchain::getBuffer(const BufferDescription& description)
 {
-    auto& device = GBMDevice::singleton();
+    auto* device = PlatformDisplay::sharedDisplayForCompositing().gbmDevice();
+    if (!device)
+        return nullptr;
```

Comment:
GPUProcess doesn't have a PlatformDisplay.

Reply:
It was functioning so far.

Tying gbm_device to PlatformDisplay was a mistake.

Reply:
How can I enable gpu process to try it out?